### PR TITLE
Dropping VMware API insecure option from Glance

### PR DESCRIPTION
**Pull request by mjura**
Because of discussion in https://github.com/crowbar/crowbar-openstack/pull/131#discussion-diff-46124162, we decided to drop it

### CODE REVIEW

**Comment by dirkmueller** [comment]
IMHO even the back implementation could be removed.. or how would you use vcenter integration without ssl on nova or cinder?

**Comment by mjura** [comment]
@dirkmueller : Glance integration can still be used independently :) anyway, I will drop this attribute completely

**Comment by mjura** [comment]
Ok, `api_secure` id dropped

**Comment by mjura** [comment]
It was rebased

**Comment by mjura** [comment]
Rebased

File: chef/data_bags/crowbar/template-glance.json [hunk 1]
```diff
@@ -73,7 +72,7 @@
     "glance": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 24,
+      "schema-revision": 23,
```
Comment (dirkmueller) [inline chef/data_bags/crowbar/template-glance.json]: thats a downgrade

Comment (mjura) [inline chef/data_bags/crowbar/template-glance.json]: Ok, I fixed it

**Comment by dirkmueller** [comment]
:+1:

**Pull request merged**